sockets/sockets.js

```diff
@@ -363,6 +363,25 @@ var actionsObj = {
 
                 // Set a cooldown for the sender until they can send another pm
                 pmmodCooldowns[senderSocket.id] = new Date();
+
+                // Create the mod log.
+                mlog = await ModLog.create({
```

Comment:
Could you please make this not await? We don't need to use mlog so we can just do ModLog.create() and let it run in the background. Don't have to wait for it to finish :). The function can then also be stripped of its async tag.